pkg/operator/controller/gatewayapi/controller.go

```diff
@@ -25,7 +27,10 @@ import (
 )
 
 const (
-	controllerName = "gatewayapi_controller"
+	controllerName                    = "gatewayapi_controller"
+	experimantalGatewayAPIGroupName   = "gateway.networking.x-k8s.io"
```

Comment:
nit: typo in variable name
```suggestion
	experimentalGatewayAPIGroupName   = "gateway.networking.x-k8s.io"
```

Reply:
Done.